framework/base/Module.php

```diff
@@ -705,4 +705,60 @@ public function afterAction($action, $result)
         $this->trigger(self::EVENT_AFTER_ACTION, $event);
         return $event->result;
     }
+
+    /**
+     * Layout find in parent module.
+     * @return array [module, layout]
+     */
+    private function findLayout()
+    {
+        $module  = $this;
+
+        while ($module !== null) {
+
+            if ($module !== null && is_string($module->layout)) {
+                return [$module, $module->layout];
+            }
+
+            $module = $module->module;
+        }
+
+        return [null, null];
+    }
+
+    /**
+     * Layout find in module.
+     * @param string $layout    Layout name
+     * @param string $layoutExt View default extension
+     * @return boolean|string
+     */
+    public function findLayoutFile($layout, $layoutExt)
+    {
+        $module = $this;
+
+        if ($layout === null) {
+            list($module, $layout) = $this->findLayout();
+        }
+
+        if (!is_string($layout)) {
+            return false;
+        }
+
+        if (Yii::isAlias($layout)) {
+            $file = Yii::getAlias($layout);
+        } else {
+            $file = $module->getLayoutPath() . DIRECTORY_SEPARATOR . ltrim($layout, '/');
```

Comment:
I think it should be one of `DIRECTORY_SEPARATOR . ltrim($layout, DIRECTORY_SEPARATOR);` or `'/' . ltrim($layout, '/')`